recipes/any-lite/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,21 @@
+#include "nonstd/any.hpp"
+
+#include <cassert>
+#include <string>
+
+using namespace nonstd;
+
+int main()
+{
+    std::string hello = "hello, world";
+
+    any var;
+
+    var =  'v' ; assert( any_cast<char>( var ) == 'v' );
+    var =   7  ; assert( any_cast<int >( var ) ==  7  );
+    var =  42L ; assert( any_cast<long>( var ) == 42L );
+    var = hello; assert( any_cast<std::string>( var ) == hello );
```

Comment:
`assert`'s might be optimized out.

Reply:
That's okay, the compiler ill evaluate them.

Reply:
Not if the preprocessor throws them out.

Reply:
@prince-chrismc don't use assert, it will not work for release build type. Try static_assert instead, or use the regular if condition.

Reply:
Done